src/actions/get_cocktail_actions/get_cocktail_action.js

```diff
@@ -0,0 +1,38 @@
+import Actions from "../../core/app_actions";
+import Axios from 'axios';
+import Router, { checkStatus, handleError } from '../../core/router.js';
+import { GET_A_COCKTAIL, GOOGLE_TOKEN_NAME, GET_A_RECOMMENDATION } from '../../constants.js';
+import cocktailStore from "../../stores/cocktail_store";
+import Cookies from "js-cookie";
+
+Actions.register(GET_A_COCKTAIL, payload => {
+  console.log('get a cocktail');
```

Comment:
❓ Are you leaving these console logs in on purpose? And if so, what is that purpose?

Reply:
ah, yes, also I will remove these

Reply:
(there's another on l 25)